### PR TITLE
Fix to issue #13: TemperatureDifference conversion

### PR DESCRIPTION
**Pull request by joseph-bs**
Added unit tests, fixed in Units.csv the conversion factor and generated
code for TemperatureDifference.cs
Note: there is a new reference in the project file, I don't know why - it is not intentional

### CODE REVIEW

File: Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj [hunk 1]
```diff
@@ -89,7 +90,9 @@
       <Name>QuantityTypes</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```
Comment (objorke) [inline Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj]: what is this?

File: Source/QuantityTypes/Quantities/QuantityTemplate.tt [hunk 2]
```diff
@@ -182,7 +182,7 @@ namespace QuantityTypes
             return value;
         }
 
-	/// <summary>
+        /// <summary>
```
Comment (objorke) [inline Source/QuantityTypes/Quantities/QuantityTemplate.tt]: thanks for fixing this!

File: Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj (hunk 1 above)
Comment (objorke) [inline Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj]: I guess it is a test project tag as described in
http://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for
https://connect.microsoft.com/VisualStudio/feedback/details/800245/vs2013rc-adds-to-vs2012-c-project-section-itemgroup

Comment (objorke) [inline Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj]: thanks for adding this!

Comment (joseph-bs) [inline Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj]: All the credit goes to Visual Studio 2013, I've done nothing ;-)
Thanks for pointing us at the explanation.

**Pull request merged**